apps/docs/components/components/ratingbutton.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: AtomLayout
+hideBreadcrumbs: true
+---
+
+# RatingButton
+
+The RatingButton component is an interactive input element specifically designed for evaluations on product pages. It provides users with the ability to select a rating by choosing an icon using either the cursor or keyboard input. The number of icons displayed and the actual icon used can be customized based on your specific requirements. The rating scale within the component operates from left to right.
+
+::: slot usage
+
+## Examples
+
+### Basic usage
+
+To make the RatingButton work you just have to provide a state.
+
+<Showcase showcase-name="RatingButton/Basic" style="min-height:250px">
+<!-- vue -->
+<<<../../preview/nuxt/pages/showcases/RatingButton/Basic.vue
+<!-- end vue -->
+<!-- react -->
+<<<../../preview/next/pages/showcases/RatingButton/Basic.tsx#source
+<!-- end react -->
+</Showcase>
+
+### Sizes
+
+RatingButton comes with three different sizes which are compatible with StorefrontUI icons. Size of the RatingButton maps to the icon size as follows:
+
+- `SfRatingButtonSize.sm` -> `SfIconSize.base`
+- `SfRatingButtonSize.base` -> `SfIconSize.lg`
+- `SfRatingButtonSize.lg` -> `SfIconSize.xl`
+
+<Showcase showcase-name="RatingButton/Sizes" style="min-height:250px">
+<!-- vue -->
+<<<../../preview/nuxt/pages/showcases/RatingButton/Sizes.vue
+<!-- end vue -->
+<!-- react -->
+<<<../../preview/next/pages/showcases/RatingButton/Sizes.tsx#source
+<!-- end react -->
+</Showcase>
+
+### Different number of icons
+
+Set any number of icons using `max` prop.
```

Comment:
Suggestion: I think we should write a bit more about why and what for this option exists here, not only what it does.
```suggestion
  ### Adjust maximum rating score value

Set any number using `max` prop to expand or shrink the number of available rating icons. For example, if you increase the maximum rating score to 10, the component will dynamically generate ten buttons, giving users a wider range of options to rate their experience.
```

WDYT @mattmaribojoc?